lib/administrate/fields/base.rb

```diff
@@ -16,6 +16,14 @@ def self.searchable?
         false
       end
 
+      def self.search_predicate(attribute, term)
```

Comment:
Unused method argument - `term`. If it's necessary, use `_` or `_term` as an argument name to indicate that it won't be used.